rollup.config.js

```diff
@@ -14,7 +14,11 @@ export default {
   plugins: [
     json(),
     resolve({ preferBuiltins: true }),
-    commonjs(),
+    commonjs({
+      ignore: [
+        'main'
+      ]
+    }),
```

Comment:
Sorry what is this for? Is it to make sure `require.main` remains there even after bundling?

Reply:
You are correct. Without it, rollup would transform the require and I would get something like 'main does not exist on ...'